README.md

```diff
@@ -1,2 +1,25 @@
-# python-lab-packages
+# pythonlab-packages
 Packages for use in Python Lab
```

Comment:
I'd love to see a little summary of why and what this package is patching, very similar to the PR description.

Reply:
Good idea!